tests/unit/testsuite/Bolt/Boltpay/TestHelper.php

```diff
@@ -10,17 +10,32 @@ class Bolt_Boltpay_TestHelper
      */
     public function addProduct($productId, $quantity)
     {
+        //Mage::init('default');
         /** @var Mage_Catalog_Model_Product $product */
-        $product = Mage::getModel('catalog/product')->load($productId);
+        $product = Bolt_Boltpay_ProductProvider::getProductById($productId);//Mage::getModel('catalog/product')->load($productId);
```

Comment:
All changes in TestHelper are unrelated to adding a new discount.  Please remove these changes from here and place in another PR.